modules/shared/src/main/scala/io/constellationnetwork/schema/swap.scala

```diff
@@ -103,13 +103,6 @@ object swap {
     implicit object OrderingInstance extends OrderBasedOrdering[AllowSpend]
   }
 
-  @derive(encoder)
-  case class AllowSpendView(
```

Comment:
It is not unused (or won't be). It's supposed to be returned in the same way we return transaction view

Reply:
You're right, I've forgotten to return this code